packages/config/src/projects/layer3s/hyperliquid.ts

```diff
@@ -0,0 +1,181 @@
+import {
+  EthereumAddress,
+  ProjectId,
+  UnixTime,
+  formatSeconds,
+} from '@l2beat/shared-pure'
+import {
+  DA_LAYERS,
+  DA_MODES,
+  OPERATOR,
+  RISK_VIEW,
+  addSentimentToDataAvailability,
+} from '../../common'
+import { REASON_FOR_BEING_OTHER } from '../../common/ReasonForBeingInOther'
+import { ProjectDiscovery } from '../../discovery/ProjectDiscovery'
+import { Badge } from '../badges'
+import { Layer3 } from './types'
+
+const discovery = new ProjectDiscovery('hyperliquid', 'arbitrum')
+
+const constructorHotAddress = EthereumAddress(
+  '0x45C3bbc5cA908ab49441FFe9742b919fC6dDb10A',
+) // also initial locker and finalizer
+
+const hotAddresses = [
+  EthereumAddress('0xEF2364dB5db6F5539Aa0bC111771a94Ee47637Fc'),
+  EthereumAddress('0xda6816df552c3f9e0FB64979fb357800d690d79B'),
+  EthereumAddress('0x58E1b0E63C905D5982324FCd9108582623b8132e'),
+  EthereumAddress('0x263294039413B96D25E4173a5F7599F8b3801504'),
+]
+
+const coldAddresses = [
+  EthereumAddress('0x8003FD297a7Aa477B746825E7A506675bF590E91'),
+  EthereumAddress('0x86d6AE3032732F27239075D77a1317989B52F628'),
+  EthereumAddress('0xE346B41B47296153A21E64D6bFc857C27874C6e7'),
+  EthereumAddress('0x5a92b4A6a525445c9B4FFf61C0db71dCfE305ede'),
+]
+
+//const validatorSetPowers = [1, 1, 1, 1]
+
+const challengePeriod = discovery.getContractValue<number>(
+  'HyperliquidBridge',
+  'disputePeriodSeconds',
+)
+
+const validatorSetSize = discovery.getContractValue<number>(
+  'HyperliquidBridge',
+  'nValidators',
+)
+
+const lockerThreshold = discovery.getContractValue<number>(
+  'HyperliquidBridge',
+  'lockerThreshold',
+)
+
+export const hyperliquid: Layer3 = {
+  type: 'layer3',
+  id: ProjectId('hyperliquid'),
+  hostChain: ProjectId('arbitrum'),
+  createdAt: new UnixTime(1734956888), // 2024-12-23T13:28:48Z
+  badges: [Badge.VM.AppChain, Badge.DA.CustomDA],
+  display: {
+    name: 'Hyperliquid',
+    slug: 'hyperliquid',
+    description:
+      'Hyperliquid is a performant exchange on Arbitrum, utilizing a custom consensus algorithm called HyperBFT.',
+    purposes: ['Exchange'],
+    category: 'Other',
+    reasonsForBeingOther: [
+      REASON_FOR_BEING_OTHER.NO_PROOFS,
+      REASON_FOR_BEING_OTHER.SMALL_DAC,
```

Comment:
wouldn't we use `REASON_FOR_BEING_OTHER.NO_DA_ORACLE` here because there are no commitments?

Reply:
yeah there is nothing so it’s basically the same, but signatures get actually verified and they can be seen as an oracle for something. idk doesn’t matter imo